yb-voyager/cmd/exportData.go

```diff
@@ -312,6 +311,20 @@ func debeziumExportData(ctx context.Context, tableList []*sqlname.SourceName, ta
 		if err != nil {
 			return fmt.Errorf("failed to determine if Oracle JDBC wallet location is set: %v", err)
 		}
+	} else if source.DBType == "yugabytedb" {
+		config.YBServers = source.DB().GetServers()
+		if startClean {
```

Comment:
What about starting exporter for the first time (when --start-clean flag is not provided)?

This code should follow code structure as below:
```
if migrationType == "CHANGES_ONLY" {
  config.YBStreamID, err = dbzm.ReadYBStreamID(exportDir)
  if err is stream id not found {
    config.YBStreamID, err = dbzm.GenerateAndStoreYBStreamID(*config)
    if err != nil {
      ...
    }
    utils.PrintAndLog("Generated YugabyteDB CDC stream-id: %s", ...)
  } else if err != nil {
    return fmt.Errorf(...)
  }

}
```